lib/Alien/Build/Plugin/Core/Setup.pm

```diff
@@ -83,6 +83,163 @@ sub _platform
   {
     $hash->{system_type} = 'unix';
   }
+
+  $hash->{cpu}{count} =
+    exists $ENV{ALIEN_CPU_COUNT} && $ENV{ALIEN_CPU_COUNT} > 0
+    ? $ENV{ALIEN_CPU_COUNT}
+    : _cpu_count();
+
+  $hash->{cpu}{arch} = _cpu_arch(\%Config);
+}
+
+# Retrieve number of available CPU cores. Adopted from
+# <https://metacpan.org/release/MARIOROY/MCE-1.879/source/lib/MCE/Util.pm#L49>
+# which is in turn adopted from Test::Smoke::Util with improvements.
+sub _cpu_count {
+  local $ENV{PATH} = "/usr/sbin:/sbin:/usr/bin:/bin:$ENV{PATH}";
```

Comment:
I don't think we execute anything on windows for this subroutine, but just to future proof please only modify the PATH here non-Windows.

Reply:
Done.